### PR TITLE
added act wrapper templates to encounter, transfers and medication di…

### PR DESCRIPTION
**Pull request by dczulada**
…spense

extract negations and remove duplicative global variables in importers
export negations in act templates

### CODE REVIEW

File: lib/health-data-standards/import/cat1/patient_importer.rb [hunk 1]
```diff
@@ -65,9 +66,13 @@ def initialize
...
 
           @section_importers[:encounters] = [generate_importer(EncounterPerformedImporter, "./cda:encounter[cda:templateId/@root = '2.16.840.1.113883.10.20.24.3.23']", '2.16.840.1.113883.3.560.1.79', 'performed'), #encounter performed
+                                             generate_importer(EncounterPerformedActImporter, nil, '2.16.840.1.113883.3.560.1.79'),
+                                             generate_importer(EncounterOrderActImporter, nil, '2.16.840.1.113883.3.560.1.83', 'ordered'),
                                              generate_importer(EncounterOrderImporter, nil, '2.16.840.1.113883.3.560.1.83', 'ordered'),
                                              generate_importer(TransferToImporter, nil, '2.16.840.1.113883.3.560.1.72'),
-                                             generate_importer(TransferFromImporter, nil, '2.16.840.1.113883.3.560.1.71')
+                                             generate_importer(TransferFromImporter, nil, '2.16.840.1.113883.3.560.1.71'),
+                                             generate_importer(TransferToActImporter, nil, '2.16.840.1.113883.3.560.1.72'),
```
Comment (edeyoung) [inline lib/health-data-standards/import/cat1/patient_importer.rb:74]: are there any issues with these having the same oids associated with them as other importers?

Comment (dczulada) [reply lib/health-data-standards/import/cat1/patient_importer.rb:74]: Both TransferFromAct and TransferFrom map to the same HQMF type "Transfer From" (2.16.840.1.113883.3.560.1.71).  This oid is just recorded in the entry.

**Comment by alexanderelliott121** [comment]
The templates look good to me.

**Comment by okeefm** [comment]
@dczulada Do we need tests for act wrapper importing?

**Review by edeyoung — COMMENTED** [review]
I approve pending O'Keefe's approval (addition of tests)

**Comment by okeefm** [comment]
Rebased against master.

**Comment by okeefm** [comment]
I added tests for:
* Encounter, Order act-based import
* Encounter, Performed act-based import
* Medication, Dispensed act-based import
* Transfer To act-based and non-act-based import
* Transfer From act-based and non-act-based import

**Review by okeefm — APPROVED** [review]
Looks good to me, now that tests are written.

**Pull request merged**